docs/development/releases-and-deployment.md

```diff
@@ -0,0 +1,102 @@
+# Spark release / deployment guidelines
+
+## GitHub releases
+
+* https://github.com/Midburn/Spark/releases
+* There should always be 1 draft release against master branch
+* This draft release is updated on each merge or commit to master.
+
+### Publishing a release
+
+* When release is ready (e.g. have enough content / is stable enough)- you can publish it
```

Comment:
"you can publish it" - who is you? Individual contributor? Release manager / team lead?
What does publishing mean? I think a paragraph explaining this process would be helpful.